log/logger.go

```diff
@@ -60,16 +60,12 @@ func (l *Logger) Warn(message string, a ...interface{}) {
 
 // Errorf checks error level is activated to write the formatted message
 func (l *Logger) Errorf(message string, a ...interface{}) {
-	if l.err == true {
-		l.Log(ConsoleColors.Red()+message+ConsoleColors.White(), a...)
-	}
+	l.Log(ConsoleColors.Red()+message+ConsoleColors.White(), a...)
```

Comment:
No related specifically to this change, but setting the console color to white after the error message (the same applies to warning, info and debug messages as well) should be replaced by restoring the default color (there is a specific ANSI color sequence for that, which is "\033[0m"), otherwise if someone uses a terminal with a default foreground color other than white, after running ops the console text color will change and in extreme cases (if the background color is also white) the terminal may not be usable anymore.

Reply:
Fixed by https://github.com/nanovms/ops/pull/1065/commits/2059dffce642578e685ac1e196bc13571ae9a91d. I have tested the changes in my terminal with various dark and light presets ... looks fine.